apps/builddao/widget/components/modals/CreateProject.jsx

```diff
@@ -0,0 +1,304 @@
+const [location, setLocation] = useState("");
+const [contributors, setDistributors] = useState("");
+const [twitter, setTwitter] = useState("");
+const [gitHub, setGitHub] = useState("");
+const [telegram, setTelegram] = useState("");
+const [website, setWebsite] = useState("");
+const [selectedTabs, setSelectedTabs] = useState(
+  new Set(tabs.filter((tab) => tab.checked).map((tab) => tab.id.toLowerCase()))
+);
+const [avatar, setAvatar] = useState("");
+const [coverImage, setCoverImage] = useState("");
+
+const handleCheckboxChange = (event) => {
+  const { id } = event.target;
+  const newSelectedTabs = new Set(selectedTabs); // Create a copy to avoid mutation
+  if (event.target.checked) {
+    newSelectedTabs.add(id.toLowerCase());
+  } else {
+    newSelectedTabs.delete(id.toLowerCase());
+  }
+  setSelectedTabs(newSelectedTabs);
+};
+
+const data = Social.keys("*/profile", "final");
+
+if (!data) {
+  return "Loading";
+}
+
+const accounts = Object.entries(data);
+
+const allMainnetAddresses = [];
+
+for (let i = 0; i < accounts.length; ++i) {
+  const accountId = accounts[i][0];
+  allMainnetAddresses.push(accountId);
+}
+
+const handleTags = (tags) => {
+  let filtered = tags.map((tag) => (tag.customOption ? tag.label : tag));
+  setTags(filtered);
+};
+
+const handleContributors = (contributors) => {
+  let filtered = contributors.map((contributor) =>
+    contributor.customOption ? contributor.label : contributor
+  );
+  setDistributors(filtered);
+};
+
+const Main = styled.div`
+  display: flex;
+  gap: 1rem;
+  padding: 1rem;
+
+  @media (max-width: 768px) {
+    flex-direction: column;
+    .lhs,
+    .rhs {
+      width: 100%;
+    }
+  }
+  .form-control {
+    background: transparent;
+  }
+  .lhs {
+    min-width: 400px;
+    > div {
+      width: 100%;
+    }
+  }
+  .rhs {
+    min-width: 400px;
+  }
+  .form-group {
+    width: 100%;
+    & > div > div.p-2 {
+      background: transparent !important;
+      border: 1px solid var(--border-color, #ccc);
+      border-width: 0px 1px 1px;
+      border-top-style: initial;
+      border-right-style: solid;
+      border-bottom-style: solid;
+      border-left-style: solid;
+      border-top-color: initial;
+      border-right-color: rgb(222, 226, 230);
+      border-bottom-color: rgb(222, 226, 230);
+      border-left-color: rgb(222, 226, 230);
+      border-image: initial;
+      border-bottom-left-radius: 0.375rem;
+      border-bottom-right-radius: 0.375rem;
+      min-height: 9em;
+      * {
+        color: var(--text-color, #fff);
+      }
+    }
+    .nav-link {
+      color: var(--text-color, #fff);
+    }
+  }
+  .form-check {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding-left: 0px;
+  }
+`;
+useEffect(() => {
+  console.log({
+    title,
+    description,
+    location,
+    contributors,
+    twitter,
+    gitHub,
+    telegram,
+    website,
+    selectedTabs,
+    tags,
+    avatar,
+    coverImage,
+  });
```

Comment:
do we need the log? can we remove this

Reply:
Yeah we don't that was just for me, when I was working on it 😁